mantle/platform/api/azure/instance.go

```diff
@@ -55,8 +56,12 @@ func (a *API) getVMParameters(name, userdata, sshkey, storageAccountURI string,
 	//             from the following: Contains an uppercase character, Contains a
 	//             lowercase character, Contains a numeric digit, Contains a special
 	//             character) Control characters are not allowed"
-	password := fmt.Sprintf("%s%s%s", "ABC&", strconv.Itoa(rand.Int()), "xyz")
-
+	n, err := rand.Int(rand.Reader, big.NewInt(999999999999999))
+	if err != nil {
+		// in case of error fall back to math/rand
+		n.SetInt64(int64(mathrand.Int()))
```

Comment:
I did the fallback here because I really didn't want to change the `getVMParameters()` function signature to start being able to pass back `err`.

Reply:
Silently falling back to insecure isn't great.  Let's just panic here.

Reply:
IIUC that means we leak resources in Azure (i.e. the resource group and a few other things have already been created), which isn't ideal.

Reply:
Hmm.  Are there reasonable runtime cases where we'd fail to read entropy though?

Reply:
doubt it - ok I'll panic()